### PR TITLE
fix typo and outdated information

### PR DESCRIPTION
**Pull request by 733737**
this will not alter the code, only the annotation

### CODE REVIEW

**Review by duncanspumpkin — APPROVED** [review]

File: src/openrct2/ride/RideRatings.cpp
Comment (733737) [reply src/openrct2/ride/RideRatings.cpp:2]: I rather wait until Gymnasiast's latest refactor is merged

Comment (duncanspumpkin) [reply src/openrct2/ride/RideRatings.cpp:2]: None of the changes touch anywhere near the copyright banner.

Comment (733737) [reply src/openrct2/ride/RideRatings.cpp:2]: ok, I thought there would be a file conflict, but I'll try to see if I can amend the code piecemeal on my end. I'll submit as 1 PR once I'm done

**Pull request merged**
